_includes/header.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>{% assign hreflang = 'en' %}{% for lang in site.languages %}{% if lang.path == page.url %}{% assign hreflang = lang.iso-alpha2 %}{% endif %}{% endfor %}
-<html hreflang="{{hreflang}}">
+<html hreflang="{{hreflang}}" dir="{{ page.dirattr }}">
```

Comment:
Can we call the property simply `direction` instead of `dirattr`? That would be clearer for most people when looking at the MD files.

```suggestion
<html hreflang="{{hreflang}}" dir="{{ page.direction }}">
```

Reply:
ok, on it :smile: